### DISCUSSION TITLE
questions about sprint command

### DISCUSSION THREAD

**Discussion by sf8193** [ankitpokhrel/jira-cli#599] [Q&A]
My teams setup is 1 project with 5 boards, each with their own sprints. 

The problem with this is when i use `jira sprint list --current` it doesn't pull up the sprint i want, it pulls up one of the other 5 boards. 

This means i need to update my sprint id manually every two weeks so i can see a good sprint view of the board i care about. Is there a way to delineate between these 5 so when i run `jira sprint list --current` it gets the board i care about?

**Comment by ankitpokhrel** [comment]
Sprint is tied to board so it should pull up sprint in the board that you have in your config. Is it that its not fetching sprint from the configured board?

**Reply by sf8193** [reply]
not it's not, i checked my config and the board is right but when i say --current it grabs the wrong one

**Reply by sf8193** [reply]
to give some more context, it does different boards when i also do --future (--previous seems to work). when i do the simple `jira sprint list` i get all boards on the project

**Reply by sf8193** [reply]
hello, just following up -- is there anything special i need to do?

**Reply by ankitpokhrel** [reply]
Hi @sf8193, I think this might perhaps be related to the way how boards are configured in your installation. Could you please run `jira sprint list --current --debug` and see what request it is sending to fetch the sprint list?

It should first send request to get active sprint from your configured board (equivalent to the curl request below) and then fetch issues from that sprint. Is the Jira API below giving you correct sprint for the configured board?

```sh
curl --request GET \
  --user "<user>:$JIRA_API_TOKEN" \
  --url 'https://<host>/rest/agile/1.0/board/1/sprint?state=active&startAt=0&maxResults=50'
```

**Reply by karampok** [reply]
Hi i have the same issue, is there any workaround? or flag I can filter out which sprint I want? Thanks
